### PR TITLE
Update contact methods for requesting help

### PR DESCRIPTION
**Pull request by brianwarner**
The Linux Foundation has added a ticketing system to track requests for project and member services. This PR adds documentation around the new processes.

Signed-off-by: Brian Warner <brian@bdwarner.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -252,5 +252,5 @@ If you run your project's social media and would like the Foundation to share or
 
 ### All other topics
 
-For any other topics which aren't covered above, please email
-[operations@openjsf.org](mailto:operations@openjsf.org).
+For any other topics which aren't covered above, please file a ticket with the
+[file a ticket with the service desk](https://members.openjsf.org).
```
Comment (Trott) [inline README.md:256]:
```suggestion
For any other topics which aren't covered above, please
[file a ticket with the service desk](https://members.openjsf.org).
```

**Review by Trott — APPROVED** [review]

**Comment by brianwarner** [comment]
Thanks for catching that, @Trott :-)

**Review by ljharb — APPROVED** [review]
oof, i always hate when things force me to use a ticketing system instead of letting me email into it :-/

**Review by mhdawson — APPROVED** [review]
LGTM

File: README.md [hunk 2]
```diff
@@ -217,9 +217,8 @@ OpenJS Foundation Collaboration spaces are able to take advantage of several ser
 
 ### Help with project infrastructure
 
-In almost all situations, the best way to get support is to email
-[operations@openjsf.org](mailto:operations@openjsf.org) with your request. For
-example:
+In almost all situations, the best way to get support is to 
+[file a ticket with The Linux Foundation](https://members.openjsf.org). For example:
```
Comment (bnb) [inline README.md:221]: It is concerning to me how disconnected this is from Foundation staff :/

Comment (brianwarner) [reply README.md:221]: If it helps, it all goes to the same person eventually. We've just been asked to adjust our processes so they're less isolated to single person's email.

**Review by joesepi — APPROVED** [review]

**Review by WaleedAshraf — APPROVED** [review]
I'm not sure if it's already there but would be good if this tool is also introduced during new projects onboarding.

**Review by achrinza — COMMENTED** [review]
Does the [Infrastructure Menu](https://github.com/openjs-foundation/cross-project-council/blob/fb580cb7a7f54c028dd521a409a0abec48d57887/project-resources/INFRASTRUCTURE_MENU.md) need to be updated as well?

**Comment by brianwarner** [comment]
Good call @achrinza, #879 should handle this.

**Pull request merged**